modules/internal/ChapelTuple.chpl

```diff
@@ -195,8 +195,8 @@ module ChapelTuple {
   //
   pragma "compiler generated"
   pragma "last resort"
-  inline proc =(ref x: _tuple,
-                pragma "intent ref maybe const formal" y: _tuple)
+  inline operator =(ref x: _tuple,
+                    pragma "intent ref maybe const formal" y: _tuple)
```

Comment:
I think this and all following operators could be methods on `_tuple` (I see that Michael commented similarly, and think you spun off an issue to this end).  The ones prior to this should remain standalone in my opinion, since they're a language-level syntax sugar for creating new tuple types.